[ERROR] 2020-08-19 16:26:09,034 | /home/saroj/frappe-bench/apps/frappe/frappe/app.py:
Site: bloomstack
Request Error
Traceback (most recent call last):
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 50, in render
    data = render_page_by_language(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 177, in render_page_by_language
    return render_page(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 193, in render_page
    return build(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 200, in build
    return build_page(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 215, in build_page
    context = get_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/context.py", line 15, in get_context
    context = get_page_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 42, in get_page_context
    page_context = make_page_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 50, in make_page_context
    context = resolve_route(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 25, in resolve_route
    context = get_page_info_from_template(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 66, in get_page_info_from_template
    app_path = frappe.get_app_path(app)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 849, in get_app_path
    return get_pymodule_path(app_name, *joins)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_pymodule_path
    return os.path.join(os.path.dirname(get_module(scrub(modulename)).__file__), *joins)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 826, in get_module
    return importlib.import_module(modulename)
  File "/home/saroj/frappe-bench/env/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'sentry'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/saroj/frappe-bench/apps/frappe/frappe/app.py", line 73, in application
    response = frappe.website.render.render()
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 80, in render
    data = render_page(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 193, in render_page
    return build(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 200, in build
    return build_page(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 215, in build_page
    context = get_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/context.py", line 15, in get_context
    context = get_page_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 42, in get_page_context
    page_context = make_page_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 50, in make_page_context
    context = resolve_route(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 25, in resolve_route
    context = get_page_info_from_template(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 66, in get_page_info_from_template
    app_path = frappe.get_app_path(app)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 849, in get_app_path
    return get_pymodule_path(app_name, *joins)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_pymodule_path
    return os.path.join(os.path.dirname(get_module(scrub(modulename)).__file__), *joins)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 826, in get_module
    return importlib.import_module(modulename)
  File "/home/saroj/frappe-bench/env/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'sentry'
[ERROR] 2020-08-19 16:26:09,258 | /home/saroj/frappe-bench/apps/frappe/frappe/utils/error.py:
New Exception collected with id: 2020-08-19 16:26:09.036015-127.0.0.1-b82
[ERROR] 2020-08-19 16:26:28,635 | /home/saroj/frappe-bench/apps/frappe/frappe/app.py:
Site: bloomstack
Request Error
Traceback (most recent call last):
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 50, in render
    data = render_page_by_language(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 177, in render_page_by_language
    return render_page(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 193, in render_page
    return build(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 200, in build
    return build_page(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 215, in build_page
    context = get_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/context.py", line 15, in get_context
    context = get_page_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 42, in get_page_context
    page_context = make_page_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 50, in make_page_context
    context = resolve_route(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 25, in resolve_route
    context = get_page_info_from_template(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 66, in get_page_info_from_template
    app_path = frappe.get_app_path(app)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 849, in get_app_path
    return get_pymodule_path(app_name, *joins)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_pymodule_path
    return os.path.join(os.path.dirname(get_module(scrub(modulename)).__file__), *joins)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 826, in get_module
    return importlib.import_module(modulename)
  File "/home/saroj/frappe-bench/env/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'sentry'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/saroj/frappe-bench/apps/frappe/frappe/app.py", line 73, in application
    response = frappe.website.render.render()
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 80, in render
    data = render_page(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 193, in render_page
    return build(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 200, in build
    return build_page(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 215, in build_page
    context = get_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/context.py", line 15, in get_context
    context = get_page_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 42, in get_page_context
    page_context = make_page_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 50, in make_page_context
    context = resolve_route(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 25, in resolve_route
    context = get_page_info_from_template(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 66, in get_page_info_from_template
    app_path = frappe.get_app_path(app)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 849, in get_app_path
    return get_pymodule_path(app_name, *joins)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_pymodule_path
    return os.path.join(os.path.dirname(get_module(scrub(modulename)).__file__), *joins)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 826, in get_module
    return importlib.import_module(modulename)
  File "/home/saroj/frappe-bench/env/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'sentry'
[ERROR] 2020-08-19 16:26:28,638 | /home/saroj/frappe-bench/apps/frappe/frappe/app.py:
Site: bloomstack
Request Error
Traceback (most recent call last):
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 50, in render
    data = render_page_by_language(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 177, in render_page_by_language
    return render_page(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 193, in render_page
    return build(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 200, in build
    return build_page(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 215, in build_page
    context = get_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/context.py", line 15, in get_context
    context = get_page_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 42, in get_page_context
    page_context = make_page_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 50, in make_page_context
    context = resolve_route(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 25, in resolve_route
    context = get_page_info_from_template(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 66, in get_page_info_from_template
    app_path = frappe.get_app_path(app)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 849, in get_app_path
    return get_pymodule_path(app_name, *joins)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_pymodule_path
    return os.path.join(os.path.dirname(get_module(scrub(modulename)).__file__), *joins)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 826, in get_module
    return importlib.import_module(modulename)
  File "/home/saroj/frappe-bench/env/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'sentry'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/saroj/frappe-bench/apps/frappe/frappe/app.py", line 73, in application
    response = frappe.website.render.render()
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 80, in render
    data = render_page(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 193, in render_page
    return build(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 200, in build
    return build_page(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 215, in build_page
    context = get_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/context.py", line 15, in get_context
    context = get_page_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 42, in get_page_context
    page_context = make_page_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 50, in make_page_context
    context = resolve_route(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 25, in resolve_route
    context = get_page_info_from_template(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 66, in get_page_info_from_template
    app_path = frappe.get_app_path(app)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 849, in get_app_path
    return get_pymodule_path(app_name, *joins)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_pymodule_path
    return os.path.join(os.path.dirname(get_module(scrub(modulename)).__file__), *joins)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 826, in get_module
    return importlib.import_module(modulename)
  File "/home/saroj/frappe-bench/env/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'sentry'
[ERROR] 2020-08-19 16:26:28,723 | /home/saroj/frappe-bench/apps/frappe/frappe/utils/error.py:
New Exception collected with id: 2020-08-19 16:26:28.639768-127.0.0.1-6d2
[ERROR] 2020-08-19 16:26:28,744 | /home/saroj/frappe-bench/apps/frappe/frappe/utils/error.py:
New Exception collected with id: 2020-08-19 16:26:28.638884-127.0.0.1-0ab
[ERROR] 2020-08-19 16:26:30,111 | /home/saroj/frappe-bench/apps/frappe/frappe/app.py:
Site: bloomstack
Request Error
Traceback (most recent call last):
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 50, in render
    data = render_page_by_language(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 177, in render_page_by_language
    return render_page(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 193, in render_page
    return build(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 200, in build
    return build_page(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 215, in build_page
    context = get_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/context.py", line 15, in get_context
    context = get_page_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 42, in get_page_context
    page_context = make_page_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 50, in make_page_context
    context = resolve_route(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 25, in resolve_route
    context = get_page_info_from_template(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 66, in get_page_info_from_template
    app_path = frappe.get_app_path(app)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 849, in get_app_path
    return get_pymodule_path(app_name, *joins)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_pymodule_path
    return os.path.join(os.path.dirname(get_module(scrub(modulename)).__file__), *joins)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 826, in get_module
    return importlib.import_module(modulename)
  File "/home/saroj/frappe-bench/env/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'sentry'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/saroj/frappe-bench/apps/frappe/frappe/app.py", line 73, in application
    response = frappe.website.render.render()
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 80, in render
    data = render_page(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 193, in render_page
    return build(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 200, in build
    return build_page(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 215, in build_page
    context = get_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/context.py", line 15, in get_context
    context = get_page_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 42, in get_page_context
    page_context = make_page_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 50, in make_page_context
    context = resolve_route(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 25, in resolve_route
    context = get_page_info_from_template(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 66, in get_page_info_from_template
    app_path = frappe.get_app_path(app)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 849, in get_app_path
    return get_pymodule_path(app_name, *joins)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_pymodule_path
    return os.path.join(os.path.dirname(get_module(scrub(modulename)).__file__), *joins)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 826, in get_module
    return importlib.import_module(modulename)
  File "/home/saroj/frappe-bench/env/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'sentry'
[ERROR] 2020-08-19 16:26:30,159 | /home/saroj/frappe-bench/apps/frappe/frappe/utils/error.py:
New Exception collected with id: 2020-08-19 16:26:30.112379-127.0.0.1-7e5
[ERROR] 2020-08-19 16:26:38,582 | /home/saroj/frappe-bench/apps/frappe/frappe/app.py:
Site: bloomstack
Request Error
Traceback (most recent call last):
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 50, in render
    data = render_page_by_language(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 177, in render_page_by_language
    return render_page(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 193, in render_page
    return build(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 200, in build
    return build_page(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 215, in build_page
    context = get_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/context.py", line 15, in get_context
    context = get_page_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 42, in get_page_context
    page_context = make_page_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 50, in make_page_context
    context = resolve_route(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 25, in resolve_route
    context = get_page_info_from_template(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 66, in get_page_info_from_template
    app_path = frappe.get_app_path(app)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 849, in get_app_path
    return get_pymodule_path(app_name, *joins)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_pymodule_path
    return os.path.join(os.path.dirname(get_module(scrub(modulename)).__file__), *joins)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 826, in get_module
    return importlib.import_module(modulename)
  File "/home/saroj/frappe-bench/env/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'sentry'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/saroj/frappe-bench/apps/frappe/frappe/app.py", line 73, in application
    response = frappe.website.render.render()
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 80, in render
    data = render_page(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 193, in render_page
    return build(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 200, in build
    return build_page(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 215, in build_page
    context = get_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/context.py", line 15, in get_context
    context = get_page_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 42, in get_page_context
    page_context = make_page_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 50, in make_page_context
    context = resolve_route(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 25, in resolve_route
    context = get_page_info_from_template(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 66, in get_page_info_from_template
    app_path = frappe.get_app_path(app)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 849, in get_app_path
    return get_pymodule_path(app_name, *joins)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_pymodule_path
    return os.path.join(os.path.dirname(get_module(scrub(modulename)).__file__), *joins)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 826, in get_module
    return importlib.import_module(modulename)
  File "/home/saroj/frappe-bench/env/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'sentry'
[ERROR] 2020-08-19 16:26:38,614 | /home/saroj/frappe-bench/apps/frappe/frappe/utils/error.py:
New Exception collected with id: 2020-08-19 16:26:38.583673-127.0.0.1-388
[ERROR] 2020-08-19 16:30:39,729 | /home/saroj/frappe-bench/apps/frappe/frappe/app.py:
Site: bloomstack
Request Error
Traceback (most recent call last):
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 50, in render
    data = render_page_by_language(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 177, in render_page_by_language
    return render_page(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 193, in render_page
    return build(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 200, in build
    return build_page(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 215, in build_page
    context = get_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/context.py", line 15, in get_context
    context = get_page_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 42, in get_page_context
    page_context = make_page_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 50, in make_page_context
    context = resolve_route(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 25, in resolve_route
    context = get_page_info_from_template(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 66, in get_page_info_from_template
    app_path = frappe.get_app_path(app)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 849, in get_app_path
    return get_pymodule_path(app_name, *joins)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_pymodule_path
    return os.path.join(os.path.dirname(get_module(scrub(modulename)).__file__), *joins)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 826, in get_module
    return importlib.import_module(modulename)
  File "/home/saroj/frappe-bench/env/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'sentry'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/saroj/frappe-bench/apps/frappe/frappe/app.py", line 73, in application
    response = frappe.website.render.render()
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 80, in render
    data = render_page(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 193, in render_page
    return build(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 200, in build
    return build_page(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 215, in build_page
    context = get_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/context.py", line 15, in get_context
    context = get_page_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 42, in get_page_context
    page_context = make_page_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 50, in make_page_context
    context = resolve_route(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 25, in resolve_route
    context = get_page_info_from_template(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 66, in get_page_info_from_template
    app_path = frappe.get_app_path(app)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 849, in get_app_path
    return get_pymodule_path(app_name, *joins)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_pymodule_path
    return os.path.join(os.path.dirname(get_module(scrub(modulename)).__file__), *joins)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 826, in get_module
    return importlib.import_module(modulename)
  File "/home/saroj/frappe-bench/env/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'sentry'
[ERROR] 2020-08-19 16:30:39,929 | /home/saroj/frappe-bench/apps/frappe/frappe/utils/error.py:
New Exception collected with id: 2020-08-19 16:30:39.772857-127.0.0.1-9a6
[ERROR] 2020-08-19 16:33:12,364 | /home/saroj/frappe-bench/apps/frappe/frappe/app.py:
Site: bloomstack
Request Error
Traceback (most recent call last):
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 50, in render
    data = render_page_by_language(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 177, in render_page_by_language
    return render_page(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 193, in render_page
    return build(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 200, in build
    return build_page(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 215, in build_page
    context = get_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/context.py", line 15, in get_context
    context = get_page_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 42, in get_page_context
    page_context = make_page_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 50, in make_page_context
    context = resolve_route(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 25, in resolve_route
    context = get_page_info_from_template(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 66, in get_page_info_from_template
    app_path = frappe.get_app_path(app)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 849, in get_app_path
    return get_pymodule_path(app_name, *joins)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_pymodule_path
    return os.path.join(os.path.dirname(get_module(scrub(modulename)).__file__), *joins)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 826, in get_module
    return importlib.import_module(modulename)
  File "/home/saroj/frappe-bench/env/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'sentry'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/saroj/frappe-bench/apps/frappe/frappe/app.py", line 73, in application
    response = frappe.website.render.render()
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 80, in render
    data = render_page(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 193, in render_page
    return build(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 200, in build
    return build_page(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 215, in build_page
    context = get_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/context.py", line 15, in get_context
    context = get_page_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 42, in get_page_context
    page_context = make_page_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 50, in make_page_context
    context = resolve_route(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 25, in resolve_route
    context = get_page_info_from_template(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 66, in get_page_info_from_template
    app_path = frappe.get_app_path(app)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 849, in get_app_path
    return get_pymodule_path(app_name, *joins)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_pymodule_path
    return os.path.join(os.path.dirname(get_module(scrub(modulename)).__file__), *joins)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 826, in get_module
    return importlib.import_module(modulename)
  File "/home/saroj/frappe-bench/env/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'sentry'
[ERROR] 2020-08-19 16:33:12,388 | /home/saroj/frappe-bench/apps/frappe/frappe/utils/error.py:
New Exception collected with id: 2020-08-19 16:33:12.364691-127.0.0.1-bb5
[ERROR] 2020-08-19 16:33:12,932 | /home/saroj/frappe-bench/apps/frappe/frappe/app.py:
Site: bloomstack
Request Error
Traceback (most recent call last):
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 50, in render
    data = render_page_by_language(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 177, in render_page_by_language
    return render_page(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 193, in render_page
    return build(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 200, in build
    return build_page(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 215, in build_page
    context = get_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/context.py", line 15, in get_context
    context = get_page_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 42, in get_page_context
    page_context = make_page_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 50, in make_page_context
    context = resolve_route(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 25, in resolve_route
    context = get_page_info_from_template(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 66, in get_page_info_from_template
    app_path = frappe.get_app_path(app)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 849, in get_app_path
    return get_pymodule_path(app_name, *joins)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_pymodule_path
    return os.path.join(os.path.dirname(get_module(scrub(modulename)).__file__), *joins)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 826, in get_module
    return importlib.import_module(modulename)
  File "/home/saroj/frappe-bench/env/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'sentry'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/saroj/frappe-bench/apps/frappe/frappe/app.py", line 73, in application
    response = frappe.website.render.render()
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 80, in render
    data = render_page(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 193, in render_page
    return build(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 200, in build
    return build_page(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 215, in build_page
    context = get_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/context.py", line 15, in get_context
    context = get_page_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 42, in get_page_context
    page_context = make_page_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 50, in make_page_context
    context = resolve_route(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 25, in resolve_route
    context = get_page_info_from_template(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 66, in get_page_info_from_template
    app_path = frappe.get_app_path(app)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 849, in get_app_path
    return get_pymodule_path(app_name, *joins)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_pymodule_path
    return os.path.join(os.path.dirname(get_module(scrub(modulename)).__file__), *joins)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 826, in get_module
    return importlib.import_module(modulename)
  File "/home/saroj/frappe-bench/env/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'sentry'
[ERROR] 2020-08-19 16:33:12,965 | /home/saroj/frappe-bench/apps/frappe/frappe/utils/error.py:
New Exception collected with id: 2020-08-19 16:33:12.934058-127.0.0.1-5e5
[ERROR] 2020-08-19 16:33:13,665 | /home/saroj/frappe-bench/apps/frappe/frappe/app.py:
Site: bloomstack
Request Error
Traceback (most recent call last):
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 50, in render
    data = render_page_by_language(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 177, in render_page_by_language
    return render_page(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 193, in render_page
    return build(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 200, in build
    return build_page(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 215, in build_page
    context = get_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/context.py", line 15, in get_context
    context = get_page_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 42, in get_page_context
    page_context = make_page_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 50, in make_page_context
    context = resolve_route(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 25, in resolve_route
    context = get_page_info_from_template(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 66, in get_page_info_from_template
    app_path = frappe.get_app_path(app)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 849, in get_app_path
    return get_pymodule_path(app_name, *joins)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_pymodule_path
    return os.path.join(os.path.dirname(get_module(scrub(modulename)).__file__), *joins)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 826, in get_module
    return importlib.import_module(modulename)
  File "/home/saroj/frappe-bench/env/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'sentry'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/saroj/frappe-bench/apps/frappe/frappe/app.py", line 73, in application
    response = frappe.website.render.render()
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 80, in render
    data = render_page(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 193, in render_page
    return build(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 200, in build
    return build_page(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 215, in build_page
    context = get_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/context.py", line 15, in get_context
    context = get_page_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 42, in get_page_context
    page_context = make_page_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 50, in make_page_context
    context = resolve_route(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 25, in resolve_route
    context = get_page_info_from_template(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 66, in get_page_info_from_template
    app_path = frappe.get_app_path(app)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 849, in get_app_path
    return get_pymodule_path(app_name, *joins)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_pymodule_path
    return os.path.join(os.path.dirname(get_module(scrub(modulename)).__file__), *joins)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 826, in get_module
    return importlib.import_module(modulename)
  File "/home/saroj/frappe-bench/env/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'sentry'
[ERROR] 2020-08-19 16:33:13,695 | /home/saroj/frappe-bench/apps/frappe/frappe/utils/error.py:
New Exception collected with id: 2020-08-19 16:33:13.666638-127.0.0.1-747
[ERROR] 2020-08-19 16:33:14,692 | /home/saroj/frappe-bench/apps/frappe/frappe/app.py:
Site: bloomstack
Request Error
Traceback (most recent call last):
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 50, in render
    data = render_page_by_language(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 177, in render_page_by_language
    return render_page(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 193, in render_page
    return build(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 200, in build
    return build_page(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 215, in build_page
    context = get_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/context.py", line 15, in get_context
    context = get_page_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 42, in get_page_context
    page_context = make_page_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 50, in make_page_context
    context = resolve_route(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 25, in resolve_route
    context = get_page_info_from_template(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 66, in get_page_info_from_template
    app_path = frappe.get_app_path(app)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 849, in get_app_path
    return get_pymodule_path(app_name, *joins)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_pymodule_path
    return os.path.join(os.path.dirname(get_module(scrub(modulename)).__file__), *joins)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 826, in get_module
    return importlib.import_module(modulename)
  File "/home/saroj/frappe-bench/env/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'sentry'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/saroj/frappe-bench/apps/frappe/frappe/app.py", line 73, in application
    response = frappe.website.render.render()
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 80, in render
    data = render_page(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 193, in render_page
    return build(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 200, in build
    return build_page(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/render.py", line 215, in build_page
    context = get_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/context.py", line 15, in get_context
    context = get_page_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 42, in get_page_context
    page_context = make_page_context(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 50, in make_page_context
    context = resolve_route(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 25, in resolve_route
    context = get_page_info_from_template(path)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/website/router.py", line 66, in get_page_info_from_template
    app_path = frappe.get_app_path(app)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 849, in get_app_path
    return get_pymodule_path(app_name, *joins)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_pymodule_path
    return os.path.join(os.path.dirname(get_module(scrub(modulename)).__file__), *joins)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 826, in get_module
    return importlib.import_module(modulename)
  File "/home/saroj/frappe-bench/env/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'sentry'
[ERROR] 2020-08-19 16:33:14,722 | /home/saroj/frappe-bench/apps/frappe/frappe/utils/error.py:
New Exception collected with id: 2020-08-19 16:33:14.692953-127.0.0.1-78a
[ERROR] 2020-08-19 16:37:09,203 | /home/saroj/frappe-bench/apps/frappe/frappe/app.py:
Site: bloomstack
Form Dict: {
 "cmd": "frappe.realtime.get_user_info"
}
Request Error
Traceback (most recent call last):
  File "/home/saroj/frappe-bench/apps/frappe/frappe/app.py", line 64, in application
    response = frappe.api.handle()
  File "/home/saroj/frappe-bench/apps/frappe/frappe/api.py", line 59, in handle
    return frappe.handler.handle()
  File "/home/saroj/frappe-bench/apps/frappe/frappe/handler.py", line 24, in handle
    data = execute_cmd(cmd)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/handler.py", line 63, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/saroj/frappe-bench/apps/frappe/frappe/__init__.py", line 1054, in call
    return fn(*args, **newargs)
TypeError: get_user_info() missing 1 required positional argument: 'sid'
[ERROR] 2020-08-19 16:37:09,465 | /home/saroj/frappe-bench/apps/frappe/frappe/utils/error.py:
New Exception collected with id: 2020-08-19 16:37:09.245029-127.0.0.1-5be
